data/resistance intro.txt

```diff
@@ -0,0 +1,1187 @@
+			`	"That it concerns recent human events, what we can say is. Events that involved in, you were," the other says.`
+			`	They offer to pay for your own meal after they pick their own, but the smell tells you to decline, as you politely leave the establishment. It seems the only way to find out what these Arachi want is to head to <destination>.`
+				accept
+
+
+
+mission "House Bebliss 2"
+	name "Old Friends"
+	description "Go to <destination>, and look for Freya to see if she has found anything worthwhile on the Pug artifacts there."
+	landing
+	to offer
+		or
+			has "House Bebliss 1-A: done"
+			has "House Bebliss 1-B: done"
+	source "Factory of Eblumab"
+	destination "Pugglemug"
+	on offer
+		conversation
+			`You follow the coordinates given to you, and land on top of a mountain, close to an estate covered by a dome similar to the one over the spaceport city.`
+			`	You are received by who you assume are Arach butlers and maids, though you can't be quite sure if they are wearing formal clothing akin to what human ones would use.`
+			`	They have you follow them through the mansion, bringing you to a room where three Arachi are waiting, initially preoccupied with some schematics and blueprints scattered over a table.`
+			`	"Captain <first> <last>!" One of them comes to greet you, after one of the servants brings their attention to you. "A rare opportunity, it is, to such an exquisite guest entertain. For a drink, would you care?" After asking that, one of the maids brings a platter with several cups, and surprisingly you notice familiar smells like coffee or tea. There are also some alcoholic beverages there, and plain water too.`
+			`	"Difficult to replicate, some of these human drinks were, but for a special guest, the least we could do, it is."`
+			choice
+				`	"Thanks, but why exactly did you want to talk to me so badly?"`
+				`	"You asked me to come here for a job. What is the job?"`
+			`	"The great House Bebliss, we represent. Maintain the local hyperspace communication network, our House does."`
+			`	Another one jumps in on the conversation, saying, "Studied hyperspace links, for centuries we have. More recently, an application of such phenomena that would help the Resistance, we have searched for."`
+			`	"Knowledge, we have acquired, of certain conflict in human space. Invaded by aliens, were you? Aliens with such technology."`
+			choice
+				`	"Yeah, they are called the Pug."`
+					goto pug
+				`	"Wait, how do you know of this?"`
+					goto how
+			label how
+			`	"Know it, the Heliarchs do, so learned about it, we have."`
+			`	"Many resources, the great Arach Houses posess. Such information, trivial it is to collect."`
+			choice
+				`	"I see. Well, yes, we were invaded by aliens known as the Pug."`
+					goto pug
+				`	"And how did the Heliarch know?"`
+			`	"In "their" space, are you not? Scanned by their ships, you were. Subtly amassed, your flight records, your charted map of the galaxy, were."`
```

Comment:
Wouldn't that contradict the part in heliarch intro where some heliarch agents look at the player's map in amazement?

Reply:
the Hely agents in that mission aren't meant to be extremely high-ranked ones/Elders, they wouldn't necessarily have the information so easily available, unlike an Arach house that could acquire that information from those who do have access to it.